src/LaunchDarkly.Client/Util.cs

```diff
@@ -14,9 +14,15 @@ internal static long GetUnixTimestampMillis(DateTime dateTime)
         internal static string ExceptionMessage(Exception e)
         {
             var msg = e.Message;
+
+            if (e is System.Threading.Tasks.TaskCanceledException)
```

Comment:
When an http request times out, we get an obscure "A task was canceled" message in this exception. This is an attempt to make it more clear what is happening.